lib/FFI/Platypus/Buffer.pm

```diff
@@ -2,10 +2,11 @@ package FFI::Platypus::Buffer;
 
 use strict;
 use warnings;
+use FFI::Platypus;
 use base qw( Exporter );
 
 our @EXPORT = qw( scalar_to_buffer buffer_to_scalar );
-our @EXPORT_OK = qw ( scalar_to_pointer );
+our @EXPORT_OK = qw ( scalar_to_pointer grow set_used_length );
```

Comment:
These functions should be documented as "Exported by requests, but not by default." or similar.

Reply:
got it.